javascript/datawire_mdk_express/.bumpversion.cfg

```diff
@@ -0,0 +1,8 @@
+[bumpversion]
+current_version = 1.0.0
+commit = True
+tag = True
+tag_name = v{new_version}
```

Comment:
Will this create a mess with tags in the mdk repo? how will we distinguish tags for mdk and datawire_mdk_express?